providers/websockets/gate/ws_data_handler.go

```diff
@@ -111,7 +111,7 @@ func (h *WebsocketDataHandler) CreateMessages(
 		market, ok := h.cfg.Market.CurrencyPairToMarketConfigs[cp.String()]
 		if !ok {
 			h.logger.Debug("market not found for currency pair", zap.String("currency_pair", cp.String()))
-			return nil, fmt.Errorf("market not found for currency pair: %s", cp.String())
+			continue
```

Comment:
Ignoring this error is the same behavior as (okx, coinbase, kraken, mexc, cryptodotcom) this provider failed to start without ignoring this error, investigation is logged [here](https://linear.app/skip/issue/BLO-932/investigate-currencypairtomarketconfigs-parse-error-in-createmessages)